### DISCUSSION TITLE
Linkerd on Redis Deployments

### DISCUSSION THREAD

**Discussion by sonicSW** [linkerd/linkerd2#7813] [Q&A] [answered]
Dear community,

maybe a silly question, do you include your Redis servers inside the service mesh or you do not inject the proxy into them?

Thank you in advance.

**Comment by adleong** [comment, marked as answer]
By default Linkerd treats port 6379 (Redis) as opaque, meaning that it will proxy the connection without attempting to parse it as HTTP.  This means that you can inject Linkerd into Redis pods.  See https://linkerd.io/2.11/features/protocol-detection/ for more details.

**Reply by sonicSW** [reply]
thank you for your information
